### DISCUSSION TITLE
Is it possible to - suppress by BOTH?

### DISCUSSION THREAD

**Discussion by Rikus-Khan** [Security-Onion-Solutions/securityonion#9082] [Unsupported Versions]
I have several suppression rules that are working fine in global.sls.  However, a question about suppressing alerts which originate and terminate in my network (multiple subnets).  I have a server running a legacy Java app which is being used by multiple subnets.  I would like to be able to suppress the 17k alerts I get within 24hrs **IF** a request comes from within my network(s) to my network.  I still want to see if any Alerts for requests going outside or coming from outside my network trying to access this server.  I see there is an option for Rate_Filter Track: by_both, but I am unsure what this does or if I can tweak it to fit my needs.  Any help or ideas would be great.

**Comment by dougburks** [comment]
Looks like you can use `threshold` or `rate_filter` with `by_both`:
https://suricata.readthedocs.io/en/suricata-6.0.0/configuration/global-thresholds.html#threshold-event-filter
